src/lib/fileHelpers.ts

```diff
@@ -80,8 +68,12 @@ export const convertFolderToModule = (
 
             edit.renameFile(e.files[0], Uri.file(uniqueModulePath));
 
-            workspace.applyEdit(edit).then(() => {
-              copySampleModuleFilesToFolder(uniqueModulePath);
+            workspace.applyEdit(edit).then(async () => {
+              await downloadGitHubRepoContents(
+                'hubspot-cli',
+                'packages/cli-assets/cms/modules/Sample.module',
+                uniqueModulePath
+              );
```

Comment:
Doesn't technically matter (same behavior) but curious on your thought process calling the download directly here as opposed to keeping `createModule` and keeping the implementation abstracted out / handled by cli-lib

Reply:
I think we have the need to download these sample assets in both the cli and here, so it makes sense to me to have the shared logic kept in `@hubspot/cli-lib` -- especially since that is already a dependency of this repo.

Reply:
Also FYI, I think this change may cause a conflict with your branch `refactor-create`. @jsines

Reply:
Yep - even if we use keep using `createModule` it'll have to be async, as will templates. I'll update mine so that once `cli-lib` changes are done it'll be good to go 👍